db-migrations/7_add_survey_type_social.up.sql

```diff
@@ -0,0 +1,10 @@
+DO $$
+BEGIN
+    IF NOT EXISTS (SELECT 1 FROM pg_type WHERE typname = 'surveytype') THEN
+        create type survey.surveytype AS ENUM ('Business', 'Social', 'Census');
+    END IF;
+END
+$$;
+ALTER TABLE survey.survey ADD COLUMN IF NOT EXISTS surveytype survey.surveytype;
+UPDATE survey.survey SET surveytype = 'Business' WHERE surveytype IS NULL;
+ALTER TABLE survey.survey ALTER COLUMN surveytype SET NOT NULL;
```

Comment:
This will break current create survey functionality in response operations. I can't find a create survey button in response operations but there is an active endpoint /surveys/create which doesn't pass survey type.

Reply:
https://github.com/ONSdigital/response-operations-ui/pull/194 should fix it @tej99. Just running tests to check it's not a breaking change with what already exists

Reply:
The 'secret url' was the favoured approach to security in the absence of ACLs so, yes, good spot 👍